src/translations/dictionary.ts

```diff
@@ -9,6 +9,10 @@ const dictionary = {
   toPlace: _('Destinasjon', 'Destination'),
   navigation: {
     assistant: _(`Reise${softhyphen}søk`, 'Assistant'),
+    assistant_a11y: _(
+      `Aktiver for reisesøk, trykk to ganger for å nullstille søket.`,
```

Comment:
Usikker på om dette fanger opp nyansene godt nok, men tror ikke vi skal være for verbos på en sa hyppig lest knapp heller.

Reply:
Kanskje det burde være 
```suggestion
      `Aktiver for reisesøk. Aktiver to ganger for å nullstille søket.`,
```